FLEDGE.md

```diff
@@ -933,16 +978,32 @@ The output of `generateBid()` contains the following fields:
     browser to select only some of the returned adComponents in order to help
     make the ad k-anonymous. See [Flexible Component Ad Selection Considering k-Anonymity](#341-flexible-component-ad-selection-considering-k-anonymity)
     for more details.
+*   selectedBuyerAndSellerReportingId: (optional) A string from the interest
+    group's ad's `selectableBuyerAndSellerReportingIds` array; if it's not in
+    the array, the bid is ignored. If present, this is reported to `reportWin()`
```

Comment:
Verifying that an empty string is permitted.

Reply:
If the empty string is member of `selectableBuyerAndSellerReportingIds`, then, yes, it's permitted.